### PR TITLE
The exception is inserted in a code block

### PR DESCRIPTION
**Pull request by madhusudhankonda**
- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md)?
- If submitting code, have you built your formula locally prior to submission with `gradle check`?
- If submitting code, is your pull request against main? Unless there is a good reason otherwise, we prefer pull requests against main and will backport as needed.
- If submitting code, have you checked that your submission is for an [OS and architecture that we support](https://www.elastic.co/support/matrix#show_os)?
- If you are submitting this code for a class then read our [policy](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#contributing-as-part-of-a-class) for that.

### CODE REVIEW

**Comment by cbuescher** [comment]
Hi @madhusudhankonda, 

thanks for the suggested change to the documentaton but I don't see a reason for making this change.
I'm going to close this but please feel free to reopen if you can explain the reasons for this change a little bit more.

**Comment by cbuescher** [comment]
Sorry for closing a bit quick, by looking at the surrounding document I think I see why you'd like to format this paragaph differently. I'm not sure though if the full paragraph really is the exception message (couldn't find that in the code yet) and how we format verbatim text like this in other places. Relabeling as something for the docs team to decide.

**Comment by cbuescher** [comment]
Also the current (8.4.2) full exception message isn't exactly whats in the docs any more. I wonder if it makes sense to includ the specific message here at all since its slightly bound to change over time. Here's what I get on 8.4.2:
```
Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [xxxx] in order to load field data by uninverting the inverted index. Note that this can use significant memory.
```

File: docs/reference/mapping/types/text.asciidoc [hunk 1]
```diff
@@ -221,9 +221,11 @@ NOTE: Reordering text fields can have an effect on <<query-dsl-match-query-phras
 aggregations, sorting, or scripting. If you try to sort, aggregate, or access
 values from a script on a `text` field, you will see this exception:
 
+```
 Fielddata is disabled on text fields by default. Set `fielddata=true` on
 `your_field_name` in order to load fielddata in memory by uninverting the
 inverted index. Note that this can however use significant memory.
+```
```
Comment (lockewritesdocs) [inline docs/reference/mapping/types/text.asciidoc:228]: As @cbuescher suggests, this text could change over time. I would instead change this to read:

>If you try to sort, aggregate, or access values from a script on a `text` field, you'll see an exception 

> indicating that field data is disabled by default on text fields. To load field data in memory, set `fielddata=true` on field name.
> 
> NOTE: Loading field data in memory can consume significant memory.

**Review by madhusudhankonda — COMMENTED** [review]
Cool - looks good

**Review by abdonpijpelink — APPROVED** [review]
LGTM. Thank you @madhusudhankonda !

**Pull request merged**